### PR TITLE
document complex tags for dropdown questions

### PR DESCRIPTION
**Pull request by btry**

### CODE REVIEW

File: source/formcreator/targets.rst [hunk 1]
```diff
@@ -129,6 +129,21 @@ The change is built from scratch. You may
 * set the urgency from a value or a question
 * choose the entity of the change among many policies
 * set the time to resolve
+
```
Comment (trasher) [inline source/formcreator/targets.rst:132]: Is this expected? I guess you'll end with a new list (ul) here

File: source/formcreator/targets.rst [hunk 2]
```diff
@@ -129,6 +129,21 @@ The change is built from scratch. You may
...
 * define the actors of the change
 * set the category of the change
 * define conditions to meet to generate the target ticket
+
+Target description templating
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Target ticket and target change both have description template. By default the template uses the tag ##FULLFORM##. This tag is a default template where all questions and sections are displayed in the same order as the form.
```
[the remark this answers is not in the document]
Comment (trasher) [reply source/formcreator/targets.rst:140]: (and same for others ;))

**Review by trasher — CHANGES_REQUESTED** [review]
see comments

File: source/formcreator/targets.rst (hunk 1 above)
Comment (btry) [reply source/formcreator/targets.rst]: no. fixed

File: source/formcreator/targets.rst [hunk 3]
```diff
@@ -132,3 +132,17 @@ The change is built from scratch. You may
...
 * define conditions to meet to generate the target ticket
+
+Target description templating
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Target ticket and target change both have description template. By default the template uses the tag ``##FULLFORM##``. This tag is a default template where all questions and sections are displayed in the same order as the form.
+
+Each question has 2 tags
+
+* ##question_x## which is replaced by the label of the question. X is the ID of the question.
```
Comment (trasher) [inline source/formcreator/targets.rst:143]:
```suggestion
* ``##question_x##`` which is replaced by the label of the question. X is the ID of the question.
```

File: source/formcreator/targets.rst [hunk 4]
```diff
@@ -132,3 +132,17 @@ The change is built from scratch. You may
...
+* ##answer_x## which is replaced by the answer to a question. X is the ID of the matching question.
```
Comment (trasher) [inline source/formcreator/targets.rst:144]:
```suggestion
* ``##answer_x##`` which is replaced by the answer to a question. X is the ID of the matching question.
```

File: source/formcreator/targets.rst [hunk 5]
```diff
@@ -132,3 +132,17 @@ The change is built from scratch. You may
...
+
+Questions of type dropdown support more complex expression like ##answer_x.name##. X still means the ID of a question. the word after the dot is the name of a property available in the search options of the itemtype of the dropdown question.
```
Comment (trasher) [inline source/formcreator/targets.rst:146]:
```suggestion
Questions of type dropdown support more complex expression like ``##answer_x.name##``. ``x`` still means the ID of a question. the word after the dot is the name of a property available in the search options of the itemtype of the dropdown question.
```

File: source/formcreator/targets.rst [hunk 6]
```diff
@@ -132,3 +132,17 @@ The change is built from scratch. You may
...
+
+As an example, consider a dropdown question shows locations. The itemtype set by the form designer is "Location". The designer can render in the target many fields of the location selected by the requester, such address, postcode, state, building, and many other data. To find them, the admin may rely on the list available in the search filters for a location.
```
Comment (AdrienClairembault) [inline source/formcreator/targets.rst:148]: I think you're missing a work here : `As an example, consider a dropdown question shows locations`
- `As an example, consider a dropdown question that shows locations` ?
- `As an example, consider a dropdown question on locations` ?

Comment (btry) [reply source/formcreator/targets.rst:148]: i'll fix it  tomorrow

File: source/formcreator/targets.rst [hunk 7]
```diff
@@ -132,3 +132,17 @@ The change is built from scratch. You may
...
+Target ticket and target change both have description template. By default the template uses the tag ``##FULLFORM##``. This tag is a default template where all questions and sections are displayed in the same order as the form.
+
+Each question has 2 tags
+
+* ``##question_x##`` which is replaced by the label of the question. X is the ID of the question.
+* ``##answer_x##`` which is replaced by the answer to a question. X is the ID of the matching question.
+
+Questions of type dropdown support more complex expression like ##answer_x.name##. X still means the ID of a question. the word after the dot is the name of a property available in the search options of the itemtype of the dropdown question.
+
+As an example, consider a dropdown question shows locations. The itemtype set by the form designer is "Location". The designer can render in the target many fields of the location selected by the requester, such address, postcode, state, building, and many other data. To find them, the admin may rely on the list available in the search filters for a location.
```
Comment (btry) [inline source/formcreator/targets.rst:148]:
```suggestion
As an example, consider a dropdown question which shows locations. The itemtype set by the form designer is "Location". The designer can render in the target many fields of the location selected by the requester, such address, postcode, state, building, and many other data. To find them, the admin may rely on the list available in the search filters for a location.
```

**Review by trasher — APPROVED** [review]

**Pull request merged**